src/frontend/hooks/server/projects.ts

```diff
@@ -147,3 +148,14 @@ export function useImportProjectConfig() {
     },
   });
 }
+
+export function useGetOwnRole() {
+  const {projectId, projectApi} = useActiveProject();
+
+  return useQuery({
+    queryKey: [OWN_ROLE, projectId],
```

Comment:
Could you use a more descriptive key besides OWN_ROLE? USER_ROLE_KEY or something?

Reply:
yeah, i think it would be good to distinguish between a user role query vs this devices role query. Maybe `THIS_USERS_ROLE_KEY`?